src/styles/blocks/main.scss

```
@@ -0,0 +1,11 @@
+* {
+  box-sizing: border-box;
```

i guess better create normalize file for *
```suggestion
  box-sizing: border-box;
  margin: 0;
```